README.md

```diff
@@ -301,6 +301,27 @@ The bundle's templates are :
 
 Sylius documentation to customize these templates is available [here](https://docs.sylius.com/en/latest/customization/template.html).
 
+## Elasticsearch entity (Document) customization
+
+In order to add a product attribute to the research you have to customize the `\MonsieurBiz\SyliusSearchPlugin\Model\Documentable\DocumentableProductTrait::convertToDocument` method.
+
+E.g. if I want to add a product attribute named `model_code` with a score of 75 I have to do this in my `\App\Entity\Product\Product`:
+
+```php
+    use DocumentableProductTrait {
+        convertToDocument as convertToDocumentTrait;
```

Comment:
I would prefer another name for the alias. Like `parentConvertToDocument` or something else, in order to avoid a suffix `Trait` which is used by traits :).

Reply:
I think that `parentConvertToDocument` is not the right name because `DocumentableProductTrait` is not the parent of `Product` but its Trait. I think `traitConvertToDocument` or `convertToDocumentTrait` are better.